client/src/components/categorical/category/index.tsx

```diff
@@ -492,7 +508,7 @@ const CategoryHeader = React.memo(
           >
             <Button
               data-testid={`colorby-${metadataField}`}
-              onClick={onColorChangeClick}
+              onClick={() => onColorChangeClick(isColorAccessor)}
```

Comment:
Slowly but surely!

Reply:
Oh actually it's a functional component! I just went ahead and `useCallback()` on it 😁 Thanks for catching this!

Reply:
Nice!